runtime/src/bank/tests.rs

```diff
@@ -268,9 +268,9 @@ fn test_bank_unix_timestamp_from_genesis() {
         genesis_config.creation_time,
         bank.unix_timestamp_from_genesis()
     );
-    let slots_per_sec = 1.0
-        / (duration_as_s(&genesis_config.poh_config.target_tick_duration)
-            * genesis_config.ticks_per_slot as f32);
+    let slots_per_sec = (genesis_config.poh_config.target_tick_duration.as_secs_f32()
+        * genesis_config.ticks_per_slot as f32)
+        .recip();
```

Comment:
The previous equation was wrong; it calculated:
```
1 / (seconds / tick) * (ticks / slot)
(ticks / second) * (ticks / slot)
ticks * ticks / second * slot
```
So, I went ahead and updated it to:
```
reciprocal( (seconds / tick) * (ticks / slot) )
reciprocal( seconds / slot )
slots / second
```
If desired, I can break it out of this PR